Sources/Controllers/Settings/NavigationSettings/OAArrivalAnnouncementViewController.m

```diff
@@ -57,7 +57,7 @@ - (EOABaseNavbarStyle)getNavbarStyle
 - (void)setupCustomLargeTitleView
 {
     NSString *text = OALocalizedString(@"announcement_time_descr");
-    CGFloat textWidth = DeviceScreenWidth - (kSidePadding + [OAUtilities getLeftMargin]) * 2;
+    CGFloat textWidth = (OAUtilities.isIPad && OAUtilities.isLandscape ? self.tableView.frame.size.width : DeviceScreenWidth) - (kSidePadding + [OAUtilities getLeftMargin]) * 2;
```

Comment:
There should not be any OAUtilities.isIPad or OAUtilities.isLandscape in such code.
Usage of `CGFloat textWidth = DeviceScreenWidth - ...` is also wrong. Width of the table or cell should be used.
This method should not contain DeviceScreenWidth or DeviceScreenHeight